hazelcast/src/main/java/com/hazelcast/nio/tcp/nonblocking/iobalancer/LoadTracker.java

```diff
@@ -138,14 +135,14 @@ private void updateNewFinalImbalance() {
         }
     }
 
-    public void notifyConnectionAdded(TcpIpConnection connection) {
-        ConnectionAddedTask connectionAddedTask = new ConnectionAddedTask(connection);
-        tasks.offer(connectionAddedTask);
+    public void notifyConnectionAdded(MigratableHandler handler) {
```

Comment:
It can be renamed from `notifyConnectionAdded ` to `notifyHandlerAdded` since we are not passing connection as a parameter any more.